packages/type/scss/font-face/_mono.scss

```diff
@@ -7,7 +7,7 @@
 
 @import '@carbon/import-once/scss/import-once';
```

Comment:
Could we remove the `import-once` import since we aren't using it?

Reply:
```suggestion
```